src/components/Popup/Index.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { FC } from "react";
+
+export const Index: FC = ({ children }) => (
```

Comment:
rename it to Popup, if there will be default it is ok to filename should be index.tsx but not with **capital letter**